common/data_refinery_common/models/models.py

```diff
@@ -784,17 +784,30 @@ def calculate_size(self) -> None:
         self.size_in_bytes = os.path.getsize(self.absolute_file_path)
         return self.size_in_bytes
 
-    def delete_local_file(self):
+    def delete_local_file(self, force=False):
         """ Deletes a file from the path and actually removes it from the file system,
         resetting the is_downloaded flag to false. Can be refetched from source if needed. """
-        if not settings.RUNNING_IN_CLOUD:
+        if not settings.RUNNING_IN_CLOUD and not force:
             return
 
         try:
             os.remove(self.absolute_file_path)
         except OSError:
             pass
```

Comment:
Not really in this PR: If the file can't be deleted, should we create a log message?

Reply:
I don't think so... at least not yet. I think there's some good reasons for this to fail, such as the file getting deleted by something else. I think that if we notice our EFS isn't getting cleaned up properly once we've addressed #452, then we could improve logging around data cleanup. I could be persuaded otherwise though.

Reply:
I see your point. I am a little worried that the failure to delete a local may indicate something serious, such as the crash of the file system. But if EFS is robust enough, then we can ignore it for now.

Reply:
I think that EFS crashes the rest of the system will be freaking out enough that it should be obvious to us that's what's happening.